ES/ESmr039.xml

```diff
@@ -145,18 +145,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <additions>
                      <list>
                         <item xml:id="a1">
-                           <locus target="#33vb">f. 33vb</locus>
-                           <desc>: Ownership note extended by an exhortation to the
-                              community.</desc>
-                           <quote xml:lang="gez">
-                              <locus target="#33vb">33vb</locus>: ዝመጽሐፈ፡ ጥምቀት፡ ዘመስጥሕ፡ ዘአባ፡ ሊባኖስ፡
-                              ዘአጽሐፍዋ፡ አቡነ፡ ወልደ፡ ተንሣኤ፡ ወቄስ፡ ገበዝ፡ አቡነ፡ ቄስ፡ ኢዮአብ፡ ምነው፡ አባቶቼ፡ ወንድሞቼ፡
-                              በምን፡ ነገር፡ ትጠሉኝ፡ አላቹህ፡ አሁን፡ አንድ፡ ቁምነገር፡ ሳልሰራ፡ አልቀርም፡ አትጥሉኝ።</quote>
-                           <quote xml:lang="en"/> Written partly in Gǝʿǝz and Amharic, possibly by
+                           <locus target="#33vb"/>
+                           <desc type="OwnershipNote">An ownership note extended with an exhortation to the
+                              community</desc>
+                           <q xml:lang="gez">
+                              ዝመጽሐፈ፡ ጥምቀት፡ ዘመስጥሕ፡ ዘአባ፡ ሊባኖስ፡
+                              ዘአጽሐፍዋ፡ አቡነ፡ ወልደ፡ ትንሣኤ፡ ወቄስ፡ ገበዝ፡ አቡነ፡ ቄስ፡ ኢዮአብ፡ ምነው፡ አባቶቼ፡ ወንድሞቼ፡
+                              በምን፡ ነገር፡ ትጠሉኝ፡ አላቹህ፡ አሁን፡ አንድ፡ ቁምነገር፡ ሳልሰራ፡ አልቀርም፡ አትጥሉኝ።</q>
+                           <note> Written partly in Gǝʿǝz and Amharic, possibly by
                            the same hand as the main text. The note mentions that the Ms. is owned
                            by the church of Libānos of Mäsṭǝḥ (cp. Additio 1 in MR-041 and MR-037).
                            It mentions also the name of the commissioners of the Ms., ʾabuna Walda
-                           Tänśaʾe and qes gabaz<note> ʾIyoʾab. </note>
+                           Tänśaʾe and qes gabaz</note> ʾIyoʾab. 
```

Comment:
If this is for future catalogue then transcriptions should be adjusted

Reply:
I wanted only to correct one Ethiopic letter, the word appears in search, it is misleading.